glean-core/tests/common/mod.rs

```diff
@@ -4,6 +4,10 @@
 
 use glean_core::Glean;
 
+// #[allow(dead_code)] is required on functions in this module as a workaround
+// for https://github.com/rust-lang/rust/issues/46379
+
+#[allow(dead_code)]
```

Comment:
We should be able to put that globally for this file:

`#![allow(dead_code)]` as the first non-comment line in the file.